CIScripts/Test/TestRunner.ps1

```diff
@@ -17,71 +17,18 @@
 . $PSScriptRoot\Tests\WindowsLinuxIntegrationTests.ps1
 
 function Run-TestScenarios {
-    Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+    Param (
+        [Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions,
+        [Parameter(Mandatory = $true)] [String] $TestConfigurationFile
+    )
 
     $Job.Step("Running all integration tests", {
 
-        $SingleSubnetNetworkConfiguration = [NetworkConfiguration] @{
-            Name = $Env:SINGLE_SUBNET_NETWORK_NAME
-            Subnets = @($Env:SINGLE_SUBNET_NETWORK_SUBNET)
-        }
-
-        $MultipleSubnetsNetworkConfiguration = [NetworkConfiguration] @{
-            Name = $Env:MULTIPLE_SUBNETS_NETWORK_NAME
-            Subnets = @($Env:MULTIPLE_SUBNETS_NETWORK_SUBNET1, $Env:MULTIPLE_SUBNETS_NETWORK_SUBNET2)
-        }
-
-        $NetworkWithPolicy1Configuration = [NetworkConfiguration] @{
-            Name = $Env:NETWORK_WITH_POLICY_1_NAME
-            Subnets = @($Env:NETWORK_WITH_POLICY_1_SUBNET)
-        }
+        . "$TestConfigurationFile"
 
-        $NetworkWithPolicy2Configuration = [NetworkConfiguration] @{
-            Name = $Env:NETWORK_WITH_POLICY_2_NAME
-            Subnets = @($Env:NETWORK_WITH_POLICY_2_SUBNET)
-        }
+        $TestConfiguration = Get-TestConfiguration
```

Comment:
So, we use a consolidation seam here (we source a file with specific implementation of functions). I'm not sure I like it tbh, it generates additional 

Why not just use the method in this PR: https://github.com/codilime/contrail-windows-ci/pull/31/files ? Then we have something like `$TestConfiguration = Import-LocalizedData -FileName TestConfiguration.psd1`.

Reply:
>  it generates additional

Additional what?

I've tried to make it work before with `Import-LocalizedData` and concluded that it's really not an appropriate way to do it, because:
1. It insists on presence of subdirectories like `en_US`.
2. The name of the function is confusing
3. This file can contain map of maps, we need nested typed objects. (Perhaps `psd1` files can contain objects. I'll need to retest that)
4. We want to load 3 similar, but different `TestConfiguration`s. The easiest way to describe "take the first test configuration and change some variables" is with code.

In general, my idea was to create an interface with 3 functions (`Get-TestConfiguration`, -`Udp` and -`WindowsLinux`) and select the provider with environment variables. You can imagine that we're just selecting implementation of this interface with a file.